internal/core/items/text.rs

```diff
@@ -928,6 +934,40 @@ impl TextInput {
         );
     }
 
+    fn select_word(self: Pin<&Self>, window_adapter: &Rc<dyn WindowAdapter>, self_rc: &ItemRc) {
+        self.move_cursor(
+            TextCursorDirection::ForwardByWord,
+            AnchorMode::MoveAnchor,
+            window_adapter,
+            self_rc,
+        );
+        self.move_cursor(
+            TextCursorDirection::BackwardByWord,
+            AnchorMode::KeepAnchor,
+            window_adapter,
+            self_rc,
+        );
```

Comment:
I think this should be inverse, i.e. after a double click (or triple click) the anchor should be on the left and the cursor on the right. Details... :)

Reply:
done thank you :-)